doc/components/adder.md

```diff
@@ -102,3 +103,30 @@ Here is an example of instantiating a `SignMagnitudeAdder`:
 
     print('${sum.value.toBigInt()}');
 ```
+
+## Compound Adder
+
+A compound carry adder is a digital circuit used for binary addition. It produces sum and sum+1 outputs.
+A trivial compound adder component [`TrivialCompoundAdder`] doesnt use any RTL code optimization.
```

Comment:
Remove brackets around names in markdown (see https://github.com/intel/rohd-hcl/blob/2433983e865e96f2e16405e6e3d7de1ba18d8613/doc/components/adder.md, the brackets remain)

Make sure to use the ticks around other things below in your doc like  "CarrySelectCompoundAdder.splitSelectAdderAlgorithm4Bit"

Reply:
done